content/installation/python/PythonOverview.md

```diff
@@ -6,9 +6,7 @@ tags: "installation python agent overview"
 
 The Contrast Python agent provides runtime protection of Django, Flask and Pyramid web applications.
 
-Version 2.3.0 of the agent introduces beta Assess functionality. This provides the ability to find vulnerabilities in applications before they can be exploited.
-
->**Note:** The beta Assess feature can be licensed in **alpha** and **eval** instances of the Contrast UI.
+Version 2.8.0 of the agent introduces Assess functionality. This provides the ability to find vulnerabilities in applications before they can be exploited.
```

Comment:
Address user directly, simplify. Does this wording work?

With Python agent version 2.8.0 you can find application vulnerabilities with Assess before they can be exploited.

Reply:
Yes that sounds great! Would it be okay to make say "Starting with...." just to be clear that it's a new feature?

Reply:
Yes